clarity_ext/clarity.py

```diff
@@ -0,0 +1,18 @@
+from genologics.lims import Lims
+from genologics.config import BASEURI, USERNAME, PASSWORD
+from genologics.entities import Process
+
+
+class ClaritySession(object):
+    """
+    A wrapper around connections to Clarity.
+    """
```

Comment:
I think it's confusing not to distinguish between current_step (instance) and current_step (id). I would prefer current_step_id as input argument.